test/openapi3/general/readOnlyWriteOnly.test.js

```diff
@@ -0,0 +1,398 @@
+'use strict';
+
+const chai = require('chai');
+const expect = chai.expect;
+const schemaValidatorGenerator = require('../../../src/index');
+const path = require('path');
+
+describe.only('oai3 - readOnly/writeOnly', function () {
+    let schema;
+    before(function () {
+        const swaggerPath = path.join(__dirname, 'readOnlyWriteOnly.yaml');
+        schema = schemaValidatorGenerator.buildSchemaSync(swaggerPath, {});
+    });
+    describe('validate readOnly in request', function () {
+        it('Should return error when request body has readOnly and required prop', function () {
+            let validator = schema['/users/required']['post'].body['application/json'];
```

Comment:
This is never rewritten so could be `const`. Same applies to many lets in this file

Reply:
changed